.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
-sudo: false
+sudo: required
```

Comment:
I think themis line can be reverted (and possibly next one). At least we should not be dependant on os